web/.htaccess

```diff
@@ -1,7 +1,6 @@
 #
 # Apache/PHP/Drupal settings:
 #
-RewriteRule ^s3fs-(css|js)/(.*)$ http://S3_BUCKET.s3-S3_REGION.amazonaws.com/public/$2 [P]
```

Comment:
@LindsayYoung We need this line; see https://github.com/18F/cf-ex-drupal8/pull/7#discussion_r274376209.

Reply:
Thanks for the explanation!

Reply:
You're welcome! If you remove this change from the PR, I believe we'll be good to go.